custom_components/tuya_local/devices/gorilla_fan_v2.yaml

```diff
@@ -0,0 +1,58 @@
+name: Gorilla Fan V2
+primary_entity:
+  entity: fan
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+      icon: mdi:ceiling-fan
```

Comment:
This should be at entity level, not inside the dp definition.  If inside a dp definition, it is expected to be inside a mapping, so that it is conditional on the dp value.  But probably it is best to avoid explicitly setting an icon. It might be better to add a new translation_key for `ceiling_fan` and set the icon there, but then you won't be able to reuse the already existing translations from fan_with_presets for the preset modes, or you'll have to duplicate them in the translation files.

Reply:
Removed explicit icon